sphinxcontrib/confluencebuilder/translator/__init__.py

```diff
@@ -0,0 +1,286 @@
+# -*- coding: utf-8 -*-
+"""
+:copyright: Copyright 2016-2020 Sphinx Confluence Builder Contributors (AUTHORS)
+:license: BSD-2-Clause (LICENSE)
+"""
+
+from ..exceptions import ConfluenceError
```

Comment:
can absolute imports be used instead?

Reply:
All relative imports related to this change request have been switched to absolute imports.

Over time, I will try to replace other relative imports in this extension.